kermit/src/commonTest/kotlin/co/touchlab/kermit/TestLogWriter.kt

```diff
@@ -12,16 +12,17 @@ package co.touchlab.kermit
 
 import kotlin.test.assertEquals
 import kotlin.test.assertTrue
+import co.touchlab.stately.collections.frozenLinkedList
 
-class TestLogger(val loggable:Severity = Severity.Verbose) : Logger() {
+class TestLogWriter(val loggable:Severity = Severity.Verbose) : LogWriter() {
     data class LogEntry(
         val severity: Severity,
         val message: String,
         val tag: String?,
         val throwable: Throwable?
     )
 
-    val _logs = mutableListOf<LogEntry>()
+    val _logs = frozenLinkedList<LogEntry>()
```

Comment:
Just to confirm my understanding, this is because the global instance will get frozen and we want to be able to modify it's writers in tests?

Reply:
Yes. I tried it with global and it failed. I do think a new collections library would be useful, so we could have non-frozen in new memory model. To discuss.